Fix PDF normalization command output

jrwdunham opened the pull request:
This changes the archivematica-fpr-admin requirement to point to a dev branch which contains a migration that fixes the Ghostscript PDF normalization FPCommand so that it states the correct output FormatVersion as PDF/A 1b (not PDF/A 1a). This will need to be updated to a new AM-fpr-admin v.1.7.7 tag/release once such is available.

This is connected to https://github.com/artefactual/archivematica/issues/1158.

sevein reviewed (approved):
Final version should be v1.7.7.

👍

jrwdunham committed:
Fix PDF normalization command output

This changes the archivematica-fpr-admin requirement to point to
archivematica-fpr-admin at version/tag v1.7.7. This version contains a
migration that fixes the Ghostscript PDF normalization FPCommand so that it
states the correct output FormatVersion as PDF/A 1b (not PDF/A 1a).

jrwdunham commented:
Cherry-picked to stable/1.7.x as 6fd0076d08357d028096b41f61d28387263e9eef
